src/js/gi/components/profile/CautionaryInformation.jsx

```diff
@@ -74,75 +83,60 @@ export class CautionaryInformation extends React.Component {
         <div className="student-complaints">
           <div className="link-header">
             <h3>
-              {it.complaints.mainCampusRollUp}&nbsp;
-              <a href="http://www.benefits.va.gov/gibill/comparison_tool/about_this_tool.asp#complaints" target="_blank">student complaints</a>
+              {+it.complaints.mainCampusRollUp}&nbsp;
+              <a href="http://www.benefits.va.gov/gibill/comparison_tool/about_this_tool.asp#complaints" target="_blank">total complaints</a>
             </h3>
             <span>
               &nbsp;(<a href="http://www.benefits.va.gov/gibill/comparison_tool/about_this_tool.asp#sourcedata" target="_blank">Source</a>)
             </span>
           </div>
         </div>
 
-        <div className="table">
-          <table className="usa-table-borderless">
-            <thead>
-              <tr>
-                <th>Complaint type</th>
-                <th>This campus</th>
-                <th>
-                  <a href="http://www.benefits.va.gov/gibill/comparison_tool/about_this_tool.asp#complaints_all_campuses" target="_blank">
-                    All campuses
-                  </a>
-                </th>
-              </tr>
-            </thead>
-            <tbody>
-              {complaints.map((c) =>
-                <TableRow key={c.description} description={c.description}
-                    thisCampus={c.thisCampus} allCampuses={c.allCampuses}/>)}
-            </tbody>
-          </table>
-        </div>
-
-        <div className="list">
-          <h4>This campus</h4>
-          {complaints.map((c) =>
-            <ListRow key={c.description} description={c.description} value={c.thisCampus}/>)}
-          <div className="row">
```

Comment:
This row for "This campus" and the corresponding one for "All campuses" in the mobile list view were duplicates of the "Student complaints" (now correctly labeled as "Total complaints") rows. So these were removed.